pombola/writeinpublic/templates/writeinpublic/person-write-draft.html

```diff
@@ -3,68 +3,372 @@
+            </a>
+          </div>
+          <div class="message-form">
+            <label for="{{ form.subject.auto_id }}" class="field-label">Subject:</label>
+            <p style="margin-bottom: 0px;">
+              {{ form.subject.errors }}
             </p>
-        </div>
-        <div class="col-sm-6">
-            <p class="form-group">
-                <label for="{{ form.author_email.auto_id }}">
-                    Your email
-                    <small>Nobody will see this, ever.</small>
-                </label>
-                {{ form.author_email.errors }}
-                {{ form.author_email }}
+            {{ form.subject }}
+            <label for="{{ form.content.auto_id }}" class="field-label">Your message:</label>
+            <p style="margin-bottom: 0px;">
+              {{ form.content.errors }}
             </p>
+            {{ form.content }}
+            <p style="display: none;" id="member-salutation">Dear {% if persons|length > 1 %}{% for person in persons %} {{ person }} {% if forloop.last %}{%else%} and {%endif%}{% endfor %}{% else %}{% for person in persons %} {{ person }} {% endfor %}{% endif %}</p>
+          </div>
+          <div class="info__expand" style="margin-bottom: 1em;">
+            <a data-w-id="00a4eab4-5de1-0aa7-1f64-345242470e01" class="info-expand__trigger w-inline-block"
+              data-toggle="collapse" href="#exemplary-messages" aria-expanded="false">
+              <div class="info-expand__icon">
+                <div class="svg-icon is--large w-embed"><svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true"
+                    focusable="false" data-prefix="fas" data-icon="info-circle"
+                    class="svg-inline--fa fa-info-circle fa-w-16" role="img" viewbox="0 0 512 512">
+                    <path fill="currentColor"
+                      d="M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z">
+                    </path>
+                  </svg></div>
+              </div>
+              <p class="info-expand__text">See examples of well written messages</p>
+              <div class="info-expand__arrow">
+                <div
+                  style="-webkit-transform:translate3d(0, 0, 0) scale3d(1, 1, 1) rotateX(0) rotateY(0) rotateZ(0deg) skew(0, 0);-moz-transform:translate3d(0, 0, 0) scale3d(1, 1, 1) rotateX(0) rotateY(0) rotateZ(0deg) skew(0, 0);-ms-transform:translate3d(0, 0, 0) scale3d(1, 1, 1) rotateX(0) rotateY(0) rotateZ(0deg) skew(0, 0);transform:translate3d(0, 0, 0) scale3d(1, 1, 1) rotateX(0) rotateY(0) rotateZ(0deg) skew(0, 0)"
```

Comment:
Let's extract these inline styles into CSS and use classes

Reply:
I removed these, they should be in SCSS now. thx